meinberlin/apps/contrib/templatetags/contrib_tags.py

```diff
@@ -122,20 +124,14 @@ def get_proper_elided_page_range(p, number, on_each_side=1, on_ends=1):
 
 @register.simple_tag
 def get_external_footer():
-    # The BO footer needs to be included here as we can't include external
-    # html in the browser. Add a cron job to update the file periodically via
-    # the get_footer management command.
-    if hasattr(settings, "BERLIN_FOOTER_URL") and settings.BERLIN_FOOTER_URL:
-        footer = cache.get("footer")
-        if footer:
-            return footer
-        filepath = settings.MEDIA_ROOT + "/" + settings.BERLIN_FOOTER_PATH
-        if not os.path.isfile(filepath):
-            has_error = management.call_command("get_footer")
-            if has_error:
-                return ""
-        with open(filepath, "r") as f:
-            footer = mark_safe(f.read())
-            cache.set("footer", footer)
-            return footer
-    return ""
+    """
+    The BO footer needs to be included here as we can't include external
+    html in the browser. Add the celery task 'periodic_footer_update' in
+    django admin periodic tasks.
+    """
+    footer = cache.get("footer", "")
+    if not footer:
+        succeeded = periodic_footer_update()
+        if succeeded:
+            footer = cache.get("footer")
+    return footer
```

Comment:
the footer needs to be marked as safe with `mark_safe()`, otherwise it will be displayed as string instead of rendered as html